game-core/src/main/java/games/strategy/engine/stats/Statistics.java

```diff
@@ -12,8 +13,6 @@
  */
 @Value
 public class Statistics {
-  private final Table<String, Round, Double> productionOfPlayerInRound = HashBasedTable.create();
-  private final Table<String, Round, Double> tuvOfPlayerInRound = HashBasedTable.create();
-  private final Table<String, Round, Double> unitsOfPlayerInRound = HashBasedTable.create();
-  private final Table<String, Round, Double> victoryCitiesOfPlayerInRound = HashBasedTable.create();
+  private final Map<OverTimeStatisticType, Table<String, Round, Double>> overTimeStatistics =
```

Comment:
Not for this PR, but for consideration as follow up perhaps - any considerations to make this data structure an object, or parts of it an object?

Reply:
What do you mean by "make it an object"?

The plan is, that this data structure will be a container also for other types of statistics. Like 

- an ordered list of battles with most units
- an ordered list of territories with most battles over the game
- …

Reply:
Essentially replace usages of this data type: ` Map<OverTimeStatisticType, Table<String, Round, Double>>`, with an object, eg: `OverTimeStatistics`

`OverTimeStatistics` would itself contain the data structure  ` Map<OverTimeStatisticType, Table<String, Round, Double>>` and would encapsulate all access to it. The holder object, beyond encapsulating the underlying data, could provide a nicer API compared to direct usage of the `Map`. It can go on to possibly make testing easier as you don't have to rely on an internal data structure nor mock objects of a more complicated API, instead can mock the one holder object that should have a stable API. In turn you can test the holder object in isolation.